### PR TITLE
feat: Product listing

### PR DESCRIPTION
**Pull request by manojncsoni007**
Screenshot (186)

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work!!

File: index.html [hunk 1]
```diff
@@ -46,39 +46,47 @@
 
     <main>
         <div class="homepage-category">
-            <a href="">
+            <a href="./pages/product-listing/product-listing.html">
                 <div class="category-item">
                     <img src="./assets/gr2.png" alt="">
```
Comment (Aditi0505) [inline index.html:51]: Please provide alt description.

File: index.html [hunk 2]
```diff
@@ -46,39 +46,47 @@
...
             </a>
-            <a href="">
-                <div class="category-item">
-                    <img src="./assets/gr2.png" alt="">
-                </div>
-            </a>
-            <a href="">
+            <a href="./pages/product-listing/product-listing.html">
                 <div class="category-item">
                     <img src="./assets/gr2.png" alt="">
```
Comment (Aditi0505) [inline index.html:59]: Please provide alt description.

File: pages/product-listing/product-listing.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jewelory</title>
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="./product-listing.css">
+    <link rel="stylesheet" href="https://aems-ui.netlify.app/aems.css" />
+    <script src="https://kit.fontawesome.com/e24bb32a8e.js" crossorigin="anonymous"></script>
+
+
+
```
Comment (Aditi0505) [inline pages/product-listing/product-listing.html:15]: Remove extra spacing.

File: pages/product-listing/product-listing.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                    <div class="space-between">
+                        <input type="checkbox" name="metal" id="filter-metal" />
+                        <label>Platinum</label>
+                    </div>
+                </div>
+                <div class="filter-group">
+                    <p class="text-m"><b>Category</b></p>
+                    <div class="space-between">
+                        <input type="checkbox" name="Category" id="filter-category" />
+                        <label>Woman's Ring</label>
```
Comment (Aditi0505) [inline pages/product-listing/product-listing.html:86]: Women's ring - spell check

**Review by Aditi0505 — COMMENTED** [review]
Left few suggestion. Please look into it. Thanks!

File: index.html (hunk 2 above)
Comment (rahul31agrawal) [inline index.html:59]: Don't leave Alt = "" blank, always fill it with some text, so we will get to know which image is missing if image doesn't loads.

File: pages/product-listing/product-listing.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jewelory</title>
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="./product-listing.css">
```
Comment (rahul31agrawal) [inline pages/product-listing/product-listing.html:10]: Try to import all the CSS files in one style.css file & link that style.css files into head tag.

File: pages/product-listing/product-listing.html (hunk 3 above)
Comment (rahul31agrawal) [reply pages/product-listing/product-listing.html:15]: you can use beautify extension to make code look beautiful.

**Review by rahul31agrawal — COMMENTED** [review]
looks good to merge

**Pull request merged**